docs/samples/browser-plugin-meetings/index.html

```diff
@@ -477,6 +477,9 @@ <h2 class="collapsible">
         <div>
           <fieldset id="meetingControlsDevices">
             <legend>Media Devices</legend>
+
+            <p class="note">NOTE: You must join the meeting to update the devices.</p>
```

Comment:
there is no technical limitation preventing this from being fixed, it's just how the sample app is currently implemented, so it would be good to raise a jira to fix this properly and allow device selection outside of the meeting

Reply:
Sure, we will raise a separate Jira for this

Reply:
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-503859